task_11/file.js

```diff
@@ -0,0 +1,2 @@
+const car = require('./app')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887247)
```suggestion
const car = require("./app")
```